src/nnet3/nnet-simple-component.cc

```diff
@@ -150,11 +170,26 @@ void DropoutComponent::Backprop(const std::string &debug_info,
 
 
 void DropoutComponent::Read(std::istream &is, bool binary) {
-  ExpectOneOrTwoTokens(is, binary, "<DropoutComponent>", "<Dim>");
-  ReadBasicType(is, binary, &dim_);
-  ExpectToken(is, binary, "<DropoutProportion>");
-  ReadBasicType(is, binary, &dropout_proportion_);
-  ExpectToken(is, binary, "</DropoutComponent>");
+  // back-compatibility code.
```

Comment:
remove this comment, it's not accurate (1st token may be missing due to how components are read in, not back compatibility).